prompting/base/prompting_miner.py

```diff
@@ -159,27 +159,37 @@ def init_wandb(self):
 
     def log_event(
         self,
+        synapse: StreamPromptingSynapse,
         timing: float,
-        prompt: str,
-        completion: str,
-        system_prompt: str,
+        messages,
+        accumulated_chunks: List[str] = [],
+        accumulated_chunks_timings: List[float] = [],
         extra_info: dict = {},
     ):
         if not getattr(self, "wandb_run", None):
             self.init_wandb()
-
+                
+        dendrite_uid = self.metagraph.hotkeys.index(synapse.dendrite.hotkey)        
         step_log = {
             "epoch_time": timing,
-            # "block": self.last_epoch_block,
-            "prompt": prompt,
-            "completion": completion,
-            "system_prompt": system_prompt,
-            "uid": self.metagraph.hotkeys.index(self.wallet.hotkey.ss58_address),
-            "stake": self.metagraph.S[self.uid].item(),
-            "trust": self.metagraph.T[self.uid].item(),
-            "incentive": self.metagraph.I[self.uid].item(),
-            "consensus": self.metagraph.C[self.uid].item(),
-            "dividends": self.metagraph.D[self.uid].item(),
+            "block": self.last_epoch_block,
```

Comment:
Is this the same function that we had problems in the past with because it kept calling the 12 second ttl cache, which created too many subtensor calls? I can't seem to find where last_epoch_block is defined.

Reply:
Great catch, that was an oversight on my end, I added after our run because I missed it in the logs (although I was able to simulate the block data manually). I don't think we have `last_epoch_block` atm, but the neuroun has `.block` which gets from ttl.

I commented this line and added a TODO to have a look into this whenever we find the opportunity